include/watchdog.h

```diff
@@ -285,7 +285,8 @@ static inline int wdt_disable(struct device *dev)
  * @param dev Pointer to the device structure for the driver instance.
  * @param cfg Pointer to timeout configuration structure.
  *
- * @retval 0 If successful.
+ * @retval channel_id Index of the channel to which the timeout was assigned.
+ *                    The value must be greater than or equal to 0.
```

Comment:
What about putting it this way:
```
@retval channel_id If successful, a non-negative value indicating the index of
                   the channel to which the timeout was assigned. This value is
                   supposed to be used as the parameter in calls to wdt_feed().
```

Reply:
and if unsuccessful?

Reply:
if unsuccessful, you have the other retvals below.

Reply:
Description updated.